src/pages/index.js

```diff
@@ -40,19 +42,61 @@ const previewImageCloseButton = document.querySelector(
 const profileEditPopup = new PopupWithForm(
   "#profile-edit-modal",
   (formData) => {
-    userInfo.setUserInfo({
-      name: formData["name"],
-      title: formData["title"],
-    });
-    profileEditPopup.close();
+    return api
+      .updateUserProfile({
+        name: formData.name,
+        about: formData.about,
+      })
+      .then(() => {
+        userInfo.setUserInfo({
+          name: formData.name,
+          about: formData.about,
+        });
+        profileEditPopup.close();
+      })
+      .catch((err) => {
+        console.error(`ERROR UPDATING USER PROFILE ${err}`);
```

Comment:
You have `this.close();` inside `PopupWithForm`


And you have `catch`  inside `PopupWithForm`


So, you don't need them in `index.js`

**Please, correct it everywhere in the code**